src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/SubmissionDispatcherImpl.java

```diff
@@ -160,12 +156,14 @@ public void listTasksInDB() throws SQLException, ParseException {
                 List<String> datasets = DatasetUtil.getSatsInOperationByYear(startingYear);
 
                 for (String dataset : datasets) {
-                	
                 	LOGGER.debug("----------------------------------------> " +  dataset);
```

Comment:
remove these "------ ..."  in the comment line (It is old but it seems a good opportunity to clean the code up)

Reply:
Ok